mtags/src/main/scala-3/scala/meta/internal/pc/AutoImportsProvider.scala

```diff
@@ -8,6 +8,7 @@ import scala.jdk.CollectionConverters.*
 
 import scala.meta.internal.mtags.MtagsEnrichments.*
 import scala.meta.internal.pc.AutoImports.*
+import scala.meta.internal.pc.completions.CompletionPos
```

Comment:
Unused import?

Reply:
Actually it's used here because `autoImport` instead of just adding import might prepend owner prefix. Like: `List` -> `ju.List`